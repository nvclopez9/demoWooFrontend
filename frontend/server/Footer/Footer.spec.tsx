import React from 'react';
import '@testing-library/react/dont-cleanup-after-each';

import { render, cleanup } from '@woographql/testing';
import { Footer } from '.';

describe('DocsNav component', () => {
  afterAll(() => cleanup());

  process.env.SITE_NAME = 'Shop Demo';
  const { baseElement } = render(
    <Footer />,
  );

  it('should render mount successfully', () => {
    expect(baseElement).toMatchSnapshot();
  });
});
