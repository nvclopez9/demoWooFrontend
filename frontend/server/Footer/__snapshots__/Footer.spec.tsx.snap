// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocsNav component should render mount successfully 1`] = `
<body>
  <div>
    <div
      class="w-full h-48 mx-auto text-background bg-foreground"
    >
      <div
        class="min-h-full h-full w-full mx-auto max-w-screen-lg flex flex-wrap justify-center pt-3 px-5 lg:px-6 lg:flex-nowrap lg:items-center"
      >
        <small
          class="flex-shrink basis-auto lg:mr-2"
        >
          <strong>
            Shop Demo
          </strong>
        </small>
        <small
          class="mb-6 basis-auto lg:mb-0"
        >
          created by Â 
          <a
            class="font-bold text-background"
            href="https://woographql.com/about"
          >
            Geoff Taylor
          </a>
        </small>
        <div
          class="flex items-center justify-center order-first lg:order-last lg:justify-end basis-full lg:basis-1/3"
        >
          <a
            aria-label="Follow me on Twitter"
            class="h-16 pr-5 text-2xl text-background icon-button"
            href="https://twitter.com/kidunot89"
          >
            <i
              aria-hidden="true"
              class="fa-brands fa-twitter"
            />
          </a>
          <a
            aria-label="Follow me on Github"
            class="h-16 pr-5 text-2xl text-background icon-button"
            href="https://github.com/kidunot89"
          >
            <i
              aria-hidden="true"
              class="fa-brands fa-github"
            />
          </a>
          <a
            aria-label="Follow me on LinkedIn"
            class="h-16 pr-5 text-2xl text-background icon-button"
            href="https://linkedin.com/in/kidunot89/"
          >
            <i
              aria-hidden="true"
              class="fa-brands fa-linkedin"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</body>
`;
